src/navigation/StackNavigation.tsx

```diff
@@ -15,9 +16,12 @@ const Navigation: React.FC = () => {
       }}
     >
       <Stack.Screen name={NavigationScreen.SIGN_IN} component={SignIn} />
-      <Stack.Screen name={NavigationScreen.HOME} component={Home} />
+      <Stack.Screen
+        name={NavigationScreen.TABS_NAV}
+        component={TabNavigation}
+      />
```

Comment:
Not sure about this. Let's try to create first component -- only for StackNavigation. Seconds component -- only for TabNavigation. And third -- main component (maybe index.ts), where we will combine both of them
```suggestion
<NavigationContainer>
    <StackNavigation />
    <TabNavigation />
</NavigationContainer>
```

Reply:
The Tab navigation should be like a prop for Stack navigation, I read that

Reply:
it should be as a child, that's the best way to do it, I guess